iOS Client/TabController/AppCoordinator.swift

```diff
@@ -43,6 +44,51 @@ class AppCoordinator: NSObject, Coordinator, UITabBarControllerDelegate, StoreSu
             self.addChildCoordinator(childCoordinator: coord)
             return coord.rootViewController
         })
+        promptForNotifications()
+    }
+    
+    func promptForNotifications() {
+        let center = UNUserNotificationCenter.current()
+        
+        center.requestAuthorization(options: [.alert, .badge, .sound]) { granted, error in
+            if granted {
+                MBStore().syncDevotions { devotions, error in
+                    if error != nil {
+                        print("Could not sync devotions and schedule devotions")
+                        print(error)
+                    } else if devotions != nil {
+                        center.delegate = self
+                        self.scheduleNotifications(devotions: devotions!)
+                    }
+                }
+            } else {
+                print(error)
+            }
+        }
+    }
+    
+    func scheduleNotifications(devotions: [MBDevotion]) {
+        print("scheduling notifications")
+        let center = UNUserNotificationCenter.current()
+        center.removeAllPendingNotificationRequests()
+        devotions.forEach { devotion in
+            guard let devotionDay = Formatters.devotionDateFormatter.date(from: devotion.date), let calendar = Formatters.calendar else {
+                return
+            }
+            var dateComponents = DateComponents()
+            dateComponents.hour = 10
+            dateComponents.minute = 30
+            dateComponents.year = calendar.component(NSCalendar.Unit.year, from: devotionDay)
+            dateComponents.month = calendar.component(NSCalendar.Unit.month, from: devotionDay)
+            dateComponents.day = calendar.component(NSCalendar.Unit.day, from: devotionDay)
+            let trigger = UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: true)
+            let content = UNMutableNotificationContent()
+            content.title = devotion.verse
+            content.body = "Read your daily devotion from Mockingbird"
```

Comment:
lol do we include a random emoji every day? 💥🎉🙃

Reply:
It might be worth subclassing UNMutableNotificationContent and setting up these default values in an `init`